packages/eds-core-react/stories/playground/DataTable.tsx

```diff
@@ -112,7 +113,42 @@ const toCellValues = (data: Data[], columns: Column[]) =>
 
 const MenuButton = ({ row }: { row: string[] }) => {
   const [anchorEl, setAnchorEl] = useState<HTMLElement>()
+  const [focus, setFocus] = useState<MenuProps['focus']>(null)
   const isOpen = Boolean(anchorEl)
+
+  const openMenu = (focus: MenuProps['focus']) => {
+    setFocus(focus)
+  }
+  const closeMenu = () => {
+    setFocus(null)
+    setAnchorEl(null)
+  }
+
+  const onKeyPress = (e: React.KeyboardEvent<HTMLButtonElement>) => {
+    const { key } = e
```

Comment:
I haven't checked but you might need a `if (key === 'tab') return` here as was needed in the stories?

Reply:
Not sure what do you mean, maybe i'm missing something, but the tab key does work.

Reply:
It does not for me until i add that line (but its 'Tab' with capital T) (I mean pressing tab should move focus to next element in table, not ne stuck on the menu button)

Reply:
I see, let me add it then.

Reply:
You were absolutely right! I didin’t see it before, i was so focused on the menu element. Thanks a lot 👍